src/main/java/org/example/utility/UtilityMethods.java

```diff
@@ -24,4 +26,8 @@ public static boolean isEven(int a) {
     public static boolean isOdd(int a) {
         return !isEven(a);
     }
+
+    public static int countNegative(int[] array) {
+        return (int) Arrays.stream(com.google.common.base.Preconditions.checkNotNull(array, "Array is null.")).filter(value -> value < 0).count();
```

Comment:
For better readability:
1. import checkNotNull using static import
2. move checkNotNull to a separate line
3. arrange stream operations so every operation is on a separate line. For example:
```
stream(array)
    .filter()
    .count() 
```